styles/globals.css

```diff
@@ -1,5 +1,26 @@
 :root {
-  --space: 1rem;
+  --ratio: 1.5;
+  --s-5: calc(var(--s-4) / var(--ratio));
```

Comment:
Are these space vars? If so maybe we could just use the word "space" instead of "s", at the cost of a few more characters it just makes these more readable in the code where they are used :grin:

Reply:
Agreed. Nice use of CSS variables. Someone's been reading up.